MoreLinq/PublicAPI/net6.0/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,5 @@
 #nullable enable
+static MoreLinq.MoreEnumerable.HasDuplicates<T>(this System.Collections.Generic.IEnumerable<T>! source) -> bool
+static MoreLinq.MoreEnumerable.HasDuplicates<T>(this System.Collections.Generic.IEnumerable<T>! source, System.Collections.Generic.IEqualityComparer<T>? comparer) -> bool
```

Comment:
Seems like my definition isn't correctly done
![image](https://github.com/morelinq/MoreLINQ/assets/14810960/544e0d60-639e-4da9-b7be-7d4325e45ff2)

Reply:
You're missing the versions that get added in Extensions.g.cs. You'll have another version of each function, which is in the extension class.